### ISSUE TITLE
Login to JED issue

### ISSUE THREAD

**Issue by rajeshnanda33** [joomla/jed-issues#95]
Hello,

I can't log in to JED.
I have changed passwords many times and trying to log in, but I can't log in.

My profile id is --> 190870
Please check. I can't manage all JED listings.

**Comment by marcobeierer** [comment]
I just tried to login and the login seems to work fine.

Have you tried the `forgot your username` and `forgot your password` function at the [login view](https://extensions.joomla.org/register/profile/edit/)?

**Comment by rajeshnanda33** [comment]
Hi Marco,
Thank you for trying to help me.

**Issue Details**
I had submitted a ticket in JED for changing my registered mail id, but unfortunately, I did a spelling mistake. So after they changed the mail id, I couldn't log in.

Then I created another account at JED and submitted another ticket and described my mistake.
And they changed back to original mail id and another username.

Now I can see the new username and I can change the password successfully.
However, when I am trying to log in, it's throwing the default error as username, password mismatch or don't have an account.

Again, I submitted the support ticket with the new account. The support team replied back, it's fixed. If the problem persists report an issue here at Github. 

Still, I can't log in.
Now, I am helpless.

Hoping a help.
Thank you.
